x/dispensation/keeper/mint_controller.go

```diff
@@ -0,0 +1,35 @@
+package keeper
+
+import (
+	clptypes "github.com/Sifchain/sifnode/x/clp/types"
+	"github.com/Sifchain/sifnode/x/dispensation/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k Keeper) SetMintController(ctx sdk.Context, mintController types.MintController) {
+	store := ctx.KVStore(k.storeKey)
+	store.Set(types.MintControllerPrefix, k.cdc.MustMarshal(&mintController))
+}
+
+func (k Keeper) GetMintController(ctx sdk.Context) types.MintController {
+	controller := types.MintController{}
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.MintControllerPrefix)
```

Comment:
It will panic 
I added some code to avoid that here, 
http://github.com/Sifchain/sifnode/blob/76c47531550a7e0e8854968ae94b68c7db31c353/x/dispensation/keeper/mint_controller.go#L17-L19

Reply:
Tanmay added test code for this